tracer/src/Datadog.Trace/Activity/ActivityLinkConverter.cs

```diff
@@ -0,0 +1,125 @@
+﻿// <copyright file="ActivityLinkConverter.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Datadog.Trace.Activity.DuckTypes;
+using Datadog.Trace.Vendors.Newtonsoft.Json;
+using Datadog.Trace.Vendors.Newtonsoft.Json.Linq;
+
+namespace Datadog.Trace.Activity;
+
+internal class ActivityLinkConverter : JsonConverter<IActivityLink>
```

Comment:
As I described in this link, we no longer need to convert the ActivityLink directly to JSON: https://github.com/DataDog/dd-trace-dotnet/pull/5019/files/0de7ff1c99d332d47659a0b353965a671bc1a849#r1619433456

Reply:
Nice!